apps/spruce/src/pages/jobLogs/utils.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  LogkeeperBuildMetadataQuery,
+  TaskTestsForJobLogsQuery,
+} from "gql/generated/types";
+import {
+  JobLogsTableTestResult,
+  LogkeeperTestResult,
+  EvergreenTestResult,
+} from "./types";
+
+const getTitle = (
+  isLogkeeper: boolean,
+  options: {
+    logkeeperBuildMetadata:
+      | LogkeeperBuildMetadataQuery["logkeeperBuildMetadata"]
+      | undefined;
+    evergreenTask: TaskTestsForJobLogsQuery["task"] | undefined;
+    groupId: string | undefined;
+  },
+) => {
+  switch (isLogkeeper) {
+    case true: {
+      const { displayName } = options.evergreenTask || {};
+      const { buildNum, builder } = options.logkeeperBuildMetadata || {};
+      if (!builder || !buildNum || !displayName) {
+        return "Job Logs";
+      }
+      return `${builder} - ${buildNum} - ${displayName}`;
+    }
+    case false: {
+      const { displayName } = options.evergreenTask || {};
+      return `Job Logs - ${displayName} - ${options.groupId}`;
```

Comment:
From an information hierarchy perspective, I don't think it makes sense to include "Job Logs" in the same heading as the relevant build info. My preference would probably be to put `${displayName} - ${options.groupId}` in an H2 or H3 below, or maybe omit "Job Logs" entirely—is it useful to know?

Reply:
Good call I updated the design and added it to the design sync agenda to get some feedback.